### PR TITLE
collecting promises which were not collected

### PR DESCRIPTION
**Pull request by bekzod**
promises were missed to be collected and resolved so collecting them, also corrected double indentation

### CODE REVIEW

File: addon/mixins/copyable.js [hunk 1]
```diff
@@ -68,28 +68,26 @@ var Copyable = Ember.Mixin.create({
...
-                  });
-
-                } else {
-                  copy.get(relName).pushObject(obj);
-                }
+            queue.push(rel.then(function(array) {
+              var resolvedCopies =
+                array.map(function(obj) {
+                  if (obj.get('copyable')) {
+                    return obj.copy(passedOptions)
```
Comment (lazybensch) [inline addon/mixins/copyable.js]: missing semicolon :)

**Comment by bekzod** [comment]
added semicolon :)

**Comment by lazybensch** [comment]
@bekzod, thx for your help on this! what was the faulty behaviour before you fixed the PromiseManyArray ? and is it possible to provide a regression test ?

**Comment by lazybensch** [comment]
ah! so the individual promises of entries of a many relation were not pushed into the queue. so you would get back a copy although not all hasmany entries are properly resolved. that most likely even messes up deep copying

**Comment by bekzod** [comment]
yes it did and some times was causing recursive/inifite loop I don't remember exact conditions for it and sorry cannot write test for it

**Comment by lazybensch** [comment]
ok thx for fixing the issue though. i will try to add a test for it.

**Comment by lazybensch** [comment]
thx again @bekzod, if you are interested #10 is a PR with a regression test i just wrote, it failed before your fix and now passes. this was a quite serious bug, i'm glad its gone. !

**Comment by bekzod** [comment]
Cool, glad that it helped :+1:

**Pull request merged**